Example/CryptoOnramp Example/CryptoOnramp Example/CryptoOnrampExampleView.swift

```diff
@@ -106,59 +106,61 @@ struct CryptoOnrampExampleView: View {
             .navigationBarTitleDisplayMode(.inline)
             .navigationDestination(for: CryptoOnrampFlowCoordinator.Route.self) { route in
                 if let coordinator {
-                    switch route {
```

Comment:
I’d recommend hiding whitespace changes to see more clearly what's changed here aside from indenting inside of the newly added `ZStack`. The `ZStack`’s purpose here is simply to group the views together to apply the two modifiers once instead of on each destination view. Intentionally [not using `Group`](https://chris.eidhof.nl/post/why-i-avoid-group/) for this.

<img width="290" height="286" alt="CleanShot 2025-10-01 at 15 05 56" src="https://github.com/user-attachments/assets/e41dd044-f4e2-49d0-a0dd-a02f33f48e9f" />